dev_guide/pod_autoscaling.adoc

```diff
@@ -45,8 +45,11 @@ The following metrics are supported by horizontal pod autoscalers:
 
 |Metric |Description
```

Comment:
A useful doc for your reference: https://github.com/DirectXMan12/kubernetes.github.io/blob/7023dc32c507f0edf8ab7278ebedd9a177a00396/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md